src/modules/HappyToast.tsx

```diff
@@ -0,0 +1,28 @@
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { useAppSelector } from '../stuff/hooks';
+
+import './HappyToast.css';
+
+export const HappyToast = () => {
+  const happyMessage = useAppSelector((state) => state.game.happyToastMessage);
+
+  if (happyMessage === undefined) {
+    return null;
+  }
+
+  return (
+    <div className={`HappyToast`}>
+      {happyMessage}
+      <FontAwesomeIcon
+        icon={faXmark}
```

Comment:
Did you notice that the X does literally nothing? Clicking anywhere clears the toast, but it looked wrong without an X.